src/panels/SkyBrowserPanel/BrowserTabs/Settings.tsx

```diff
@@ -1,54 +1,114 @@
 
 interface Props {
-  id: string | undefined;
+  id: string;
 }
 
 export function Settings({ id }: Props) {
   const luaApi = useOpenSpaceApi();
-  const selectedBrowserId = useAppSelector((state) => state.skybrowser.selectedBrowserId);
-  const color = useBrowserColorString(selectedBrowserId);
-  const browserIds = useSkyBrowserIds();
+  const targetId = useAppSelector((state) => state.skybrowser.browsers[id]?.targetId);
+  const color = useBrowserColorString(id);
+  const fov = useBrowserFov(id);
+  const radius = useBrowserRadius(id);
+  const { ra, dec } = useBrowserCoords(id);
 
   function setColor(newColor: string) {
     const parsedColor = newColor.match(/\d+/g)?.map(Number);
     if (!parsedColor) {
       return;
     }
     const [r, g, b] = parsedColor;
-    luaApi?.skybrowser.setBorderColor(selectedBrowserId, r, g, b);
+    luaApi?.skybrowser.setBorderColor(id, r, g, b);
   }
 
-  function deleteBrowser() {
-    if (!id) {
-      return;
-    }
-    // If there are more browsers, select another browser
-    if (browserIds.length > 1) {
-      const otherBrowsers = browserIds.filter((b) => b !== id);
-      luaApi?.skybrowser.setSelectedBrowser(otherBrowsers[0]);
-    }
-    luaApi?.skybrowser.removeTargetBrowserPair(id);
+  function setBorderRadius(newValue: number): void {
+    luaApi?.skybrowser.setBorderRadius(id, newValue);
+  }
+
+  function setVerticalFov(newValue: number): void {
+    luaApi?.skybrowser.setVerticalFov(id, newValue);
+  }
+
+  function setRightAscension(newValue: number): void {
+    luaApi?.skybrowser.setEquatorialAim(id, newValue, dec);
+  }
+
+  function setDeclination(newValue: number): void {
+    luaApi?.skybrowser.setEquatorialAim(id, ra, newValue);
   }
 
   return (
     <Stack gap={5} my={'lg'}>
       <Title order={2}>Settings</Title>
+      <NumericInput value={radius} label={'Border Radius'} onEnter={setBorderRadius} />
       <ColorInput
         label={'Color'}
         placeholder={'Set browser color...'}
         format={'rgb'}
         defaultValue={color}
         onChange={setColor}
       />
-      <Button variant={'outline'} color={'red'} onClick={deleteBrowser} mt={'lg'}>
-        Delete browser
-      </Button>
+      <NumericInput
+        value={fov}
+        label={'Vertical field of view'}
+        min={0.01}
+        max={70.0}
+        onEnter={setVerticalFov}
+      />
+      <NumericInput
+        value={ra}
+        label={'Right Ascension'}
+        min={0}
+        max={360}
+        onEnter={setRightAscension}
+      />
+      <NumericInput
+        value={dec}
+        label={'Declination'}
+        min={-90}
+        max={90}
+        onEnter={setDeclination}
+      />
+      <Property uri={`Scene.${targetId}.Renderable.ApplyRoll`} />
+      <Property uri={`ScreenSpace.${id}.PointSpacecraft`} />
+      <Collapsable title={'Display Copies'}>
+        <SettingsDisplayCopies id={id} />
+      </Collapsable>
+      <Collapsable title={'General Settings'}>
```

Comment:
We need to add some hierarchy for the different sections here. First, use titles for the collapsables. 

See other code suggestions for ordering - these will be based on that this change is done
```suggestion
      <Collapsable title={<Title order={3}>Display Copies</Title>}>
        <SettingsDisplayCopies id={id} />
      </Collapsable>
      <Collapsable title={<Title order={3}>General Settings</Title>}>
```